code/game/objects/items/devices/radio/headset.dm

```diff
@@ -137,6 +137,12 @@
 	user.set_interaction(src)
 	tgui_interact(user)
 
+/obj/item/device/radio/headset/examine(mob/user as mob)
+	if(ishuman(user) && loc == user)
+		user.set_interaction(src)
```

Comment:
i wouldn't really worry about this, interactee stuff is mostly dead code

```suggestion
```

Reply:
done